src/components/dialogs/EditClass.tsx

```diff
@@ -88,16 +92,18 @@ const EditClassDialog = ({
   }
 
   async function handleAdd() {
+    // console.log(classItem);
+
     const classroom: Class = {
-      id: 0, // this need to be something else when editting
+      id: classItem?.id || 0,
       number: form.number,
       year: form.year,
       semester: form.semester,
       students: form.students,
       classAdvisors: form.classAdvisors,
       contacts: form.contacts,
       schedule: {
-        id: 0,
+        id: classItem?.schedule.id || 0,
         content: [], // this need to be something else when editting
       },
       subjects: [], // this need to be something else when editting
```

Comment:
But you’re doing editing right now…?

Reply:
Yes but you can't change the designated schedule for a class anyways but at most you might be able to change the content but that's not supported yet